### PR TITLE
(fix)[TYPO]Readme.md/OSS-and-Dependecy-management

### PR DESCRIPTION
**Pull request by totopsy**

### CODE REVIEW

**Comment by sottlmarek** [comment]
Please resolve the conflicts with master branch. There are several other typo fixes done in past hour. Thank you.

**Comment by totopsy** [comment]
Hello @sottlmarek.

I resolved the conflict with `master`.

Thanks

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ Secrets management includes managing, versioning, encrypting, discovery, rotatin
 
 ## OSS and Dependecy management
 
-Dependecny security testing and analysis is very important part of discovering supply chain attacks. SBOM creation and following dependency scanning (Software composition analysis) is critical part of Continuous integration. Data series and data trends tracking should be part of CI tooling. You need to know what you produce and what you consume in context of libraries and packages. 
+Dependency security testing and analysis is very important part of disocvering supply chain attacks. SBOM creation and following depenceny scanning (Software composition analysis) is critical part of Continuous integration. Data series and data trends tracking should be part of CI tooling. You need to know what you produce and what you consume in context of libraries and packages.
```
Comment (sottlmarek) [inline README.md:101]: Can you please fix the newely added typos ? Discovering and following dependency.

**Review by sottlmarek — COMMENTED** [review]
I recommend to add fixes for new 2 typos.

File: README.md (hunk 1 above)
Comment (totopsy) [reply README.md]: Strange, I don't get them. 

Is it ok now ?

**Review by sottlmarek — APPROVED** [review]
Looks good, thanks for the effort.

**Pull request merged**